src/annotator/test/sidebar-test.coffee

```diff
@@ -331,6 +365,12 @@ describe 'Sidebar', ->
       sidebar.show()
       assert.isFalse sidebar.visibleHighlights
 
+    it 'updates the toolbar', ->
```

Comment:
Could this test description (and an identical one for `#hide`) be a little clearer?